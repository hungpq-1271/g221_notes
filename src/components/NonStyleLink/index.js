import React from 'react'
import styled from 'styled-components'

const StyledLink = styled.a`
  text-decoration: none;
  color: inherit;
  cursor: pointer;
`

export default function NonStyleLink({ to, children }) {
  return <StyledLink href={to}>{children}</StyledLink>
}
