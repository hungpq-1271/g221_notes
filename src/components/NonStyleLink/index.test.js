import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'

import NonStyleLink from '../NonStyleLink'

describe('<NonStyleLink />', () => {
  it('should render and match the snapshot', () => {
    const { container } = render(<NonStyleLink />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
