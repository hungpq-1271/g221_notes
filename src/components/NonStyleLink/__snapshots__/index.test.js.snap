// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NonStyleLink /> should render and match the snapshot 1`] = `
<a
  class="sc-bdVaJa fxslth"
/>
`;
